Avoid "we" except in the acknowledgements. (#15)
cbor-wg/update-8610-grammar: draft-ietf-cbor-update-8610-grammar.md

(Notes:
In ABNF, strings such as `"A"`, `"B"` etc. are case-insensitive, as is
intended here.

> The rules above could have written `%x62` as `%s"b"` etc., but didn't, in order to

Reviewer: ```suggestion
The rules above could, instead of `%x62`, also have used `%s"b"` etc., but didn't, in order to
```

While "the rules could have written" is arguably correct, reading it gave me a kneejerk "could have … *been* written" reaction that stopped my reading flow. I'm not sure my suggestion is a real improvement – maybe you find a better rephrasing.
